### PR TITLE
[Networking] Minor mock generation cleanup

### PR DESCRIPTION
**Pull request by SaveTheRbtz**
In this diff:
* `network/mocknetwork/engine.go` was generated twice: once by `mockgen` and the other time by `mockery`
* `SubscriptionManager` mock was generated twice by mockery
* `connector_option` mock was a stale left over.

To fully eliminate "stale mock" problem we would need to move all mocks to a single directory and remove it before we generate anything.

### CODE REVIEW

**Review by pattyshack — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -149,7 +148,6 @@ generate-mocks:
 	mockery --name '.*' --dir=fvm --case=underscore --output="./fvm/mock" --outpkg="mock"
 	mockery --name '.*' --dir=fvm/state --case=underscore --output="./fvm/mock/state" --outpkg="mock"
 	mockery --name '.*' --dir=ledger --case=underscore --output="./ledger/mock" --outpkg="mock"
-	mockery --name 'SubscriptionManager' --dir=network/ --case=underscore --output="./network/mocknetwork" --outpkg="mocknetwork"
```
Comment (gomisha) [inline Makefile:152]: where is the 2nd generation of this mock? I'm looking for another instance of `mockery --name 'SubscriptionManager'`

Comment (SaveTheRbtz) [reply Makefile:152]: `mockery` generates all the mocks for `./network` down there:

```
mockery --name '.*' --dir=network --case=underscore --output="./network/mocknetwork" --outpkg="mocknetwork"
```

Comment (gomisha) [reply Makefile:152]: Thanks for the clarification.

**Review by gomisha — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Pull request merged**